src/lib/components/Picker.react.js

```diff
@@ -50,7 +52,10 @@ export default class Picker extends Component<Props, State> {
   }
 
   handleChange = (event) => {
-    const selectedDate = event
+    let selectedDate = event
+    if (selectedDate === null)
+      selectedDate = defaultProps.value;
```

Comment:
On this handling change function, would the behavior be to always return back a date? Do you think a better user experience would be to allow nulls and prevent the callback from firing until it matches the input format?

Ex: when fully clearing out the field to then paste a new date, what should happen? Reading this it appears that it will instantly update with a hard coded-date that may not be relevant to the dataset.

Reply:
Thanks for the suggestion here. I agree that allowing an empty value would make things easier for a user, so I've pushed that change. Unfortunately, I haven't found a way so far to prevent the callback from firing, so the Python code will need to be able to handle `None` values.